.gitignore

```diff
@@ -6,3 +6,5 @@
 venv/
 .pytest_cache/
 **__pycache__/
+.venv/
```

Comment:
The entry `.venv/` is redundant because `venv/` is already specified on line 6. Consider removing one of these entries to avoid redundancy.